test/Groups/quotients.jl

```diff
@@ -92,11 +92,11 @@ end
    x,y = gens(F)
    @test x == F[1]
    @test y == F[2]
-   @test is_full_fp_group(F)
+   @test Oscar._is_full_fp_group(F)
```

Comment:
It seems this needs
```suggestion
   @test Oscar._is_full_fp_group(GapObj(F))
```
here and elsewhere -- or alternatively another method which does that for you?

Reply:
Yes, these were changes in the last moment, after testing other changes ...